File: BleModule/src/main/java/com/topdon/ble/EasyBLEException.java
package com.topdon.ble;

/**
 * date: 2021/8/12 12:08
 * author: bichuanfeng
 */
public class EasyBLEException extends RuntimeException {
    private static final long serialVersionUID = -7775315841108791634L;

    public EasyBLEException(String message) {
        super(message);
    }

    public EasyBLEException(String message, Throwable cause) {
        super(message, cause);
    }

    public EasyBLEException(Throwable cause) {
        super(cause);
    }
}

File: ble-core/src/main/java/com/mpdc4gsr/ble/core/EasyBLEException.kt
package com.mpdc4gsr.ble.core

class EasyBLEException : RuntimeException {
    constructor(message: String?) : super(message)

    constructor(message: String?, cause: Throwable?) : super(message, cause)

    constructor(cause: Throwable?) : super(cause)

    companion object {
        private val serialVersionUID = -7775315841108791634L
    }
}